spec/features/create_game_spec.rb

```diff
@@ -0,0 +1,181 @@
+require 'rails_helper'
+
+RSpec.describe 'games/create' do
+  let!(:players) do
+    create_list(:player, 6) do |player, index|
+      unique_id = index + 1
+      player.id = unique_id
+      player.name = "player#{unique_id}"
+      player.save
+    end
+  end
+
+  let!(:decks) do
+    create_list(:deck, 6) do |deck, index|
+      unique_id = index + 1
```

Comment:
`create` will already generate unique ids from writing the deck in the database. No need to generate an id + assign it to the `deck`.